### DISCUSSION TITLE
Model / Resource / Policy generator

### DISCUSSION THREAD

**Discussion by aurawindsurfing** [filamentphp/filament#5542] [Ideas]
Hey,

I find myself constantly doing:

php artisan make:Model -m
php artisan make:ModelPolicy --model=Model
php artisan make:filament-resource Model

It would be really cool to be able to 

php artisan make:filament-resource Model -model -migration -policy -resource

🤯🚀🚀 

What do you guys think?

**Comment by danharrin** [comment]
sure!
